lib/index.js

```diff
@@ -249,16 +291,48 @@ module.exports = (secp256k1) => {
       }
     },
 
-    ecdsaSign (msg32, seckey, options = {}, output) {
+    ecdsaSign (msg32, seckey, options = {}, output, forceLegacy) {
```

Comment:
are we forced to change the public API here?

Reply:
If we want to have `recid` then yes

Reply:
why can't we do this instead?
```js
get recid() {
  return secp256k1.ecdsaSign(obj, msg32, seckey, options.data, options.noncefn)
}
```

Reply:
Didn't think it through, I guess :D